src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.SymbolMoniker.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.SymbolMonikers;
+using Microsoft.CodeAnalysis.FindUsages;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Shared.Utilities;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.FindUsages
+{
+    using static FindUsagesHelpers;
+
+    internal abstract partial class AbstractFindUsagesService
+    {
+        private static async Task FindSymbolMonikerReferencesAsync(
+            IFindSymbolMonikerUsagesService monikerUsagesService,
+            ISymbol definition,
+            IFindUsagesContext context,
+            CancellationToken cancellationToken)
+        {
+            if (monikerUsagesService == null)
+                return;
+
+            var moniker = SymbolMoniker.TryCreate(definition);
+            if (moniker == null)
+                return;
+
+            var progress = new StreamingProgressTracker(context.ReportProgressAsync);
+            try
+            {
+                await progress.AddItemsAsync(1).ConfigureAwait(false);
+
+                // TODO: loc this
+                var displayParts = GetDisplayParts(definition).Add(
+                    new TaggedText(TextTags.Text, " - (external)"));
+
+                var definitionItem = DefinitionItem.CreateNonNavigableItem(
+                    GlyphTags.GetTags(definition.GetGlyph()),
+                    displayParts,
+                    originationParts: ImmutableArray.Create(new TaggedText(TextTags.Text, "external")));
```

Comment:
AFAICT, OriginParts isn't ever used (outside of tests).  but i need to confirm that for sure.